pkg/tests/path_test.bzl

```diff
@@ -0,0 +1,86 @@
+# Copyright 2021 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for path.bzl"""
+
+load("@bazel_skylib//lib:unittest.bzl", "analysistest", "asserts", "unittest")
+load("//:mappings.bzl", "pkg_mkdirs")
+load("//:path.bzl", "compute_data_path")
+
+##########
+# Test compute_data_path
+##########
```

Comment:
I get why you're using analysis tests here, but it feels like much of this could instead be a unit test instead.

Might be worth a comment, but it's not essential.

Reply:
Hmmm.. I wonder if I can do that.  I will come back to that later this week if I have time.
For now, I want to get this refactoring in so that I can mail you the PR that adds pkg_files support to pkg_tar.
I pulled this out of that.